docs/agent/changelog.md

```diff
@@ -6,6 +6,20 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.15.0 - \[2024-08-29\]
+
+- Added `--url` agent CLI flag for simpler creation of HTTP(S), TLS, and TCP protocol endpoints.
+- Deprecated `--domain`, `--scheme`, and `--remote-addr` agent CLI flags, which have been replaced by `--url`.
+- Added `--metadata` and `--description` agent CLI flags when using `--url`.
+
+##### A new agent configuration format added to V3
+
+- Added `agent:` as a toplevel field to agent config, with `endpoints:` as a nested field used for [endpoint creation](/docs/agent/config/v3/#full-example).
```

Comment:
endpoints is not nested underneath agent

```
version: 3
agent:
  ...
tunnels:
  foo:
    ...
endpoints:
  - name: foo
    ...
    
```

Reply:
Bad sentence merge here, ugh.